scrnaseq.Rmd

```diff
@@ -503,10 +506,12 @@ sc = purrr::map(list_indices(sc), function(i){
   Seurat::RenameCells(sc[[i]], new.names=paste(cell_names, i, sep="-"))
 })
 
-# Set up colors for samples
-sample_names = purrr::flatten_chr(purrr::map(sc, function(s){ unique(as.character(s[[]][["orig.ident"]])) }))
+# Set up colors for samples and add to the sc objects
+sample_names = purrr::flatten_chr(purrr::map(sc, function(s) { return(unique(as.character(s[[]][["orig.ident"]]))) }))
```

Comment:
Multiple lines, those long lines with so many `([` are difficult to read ...

Reply:
done